src/azanium/pseudoace.py

```diff
@@ -21,13 +22,30 @@ def run_pseudoace(context, *args):
     logger.info(out)
 
 
+def source_annotated_models_file(context):
+    """Sources the annotated models file from github.
+
+    Returns the local filename. """
+    # Get the annotated models file separately from github.
+    gh_repo_path = 'WormBase/wormbase-pipeline'
+    gh_file_path = 'wspec/models.wrm.annot'
+    annot_file_content = github.read_released_file(gh_repo_path,
+                                                   gh_file_path)
+    version = context.data_release_version
+    target_dir = context.base_path
+    am_local_filename = os.path.basename(gh_file_path) + '.' + version
+    am_local_path = os.path.join(target_dir, am_local_filename)
+    with open(am_local_path, mode='w') as fp:
```

Comment:
The `mode` here should `wb` to avoid potential encoding issues (github3`s `.decoded` API on `file_contents` returns `bytes`